submitted_models/marble_hd2_sensor_config_1/launch/spawner.rb

```diff
@@ -0,0 +1,76 @@
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
+  <<-HEREDOC
+  <spawn name='#{_name}'>
+    <name>#{_name}</name>
+    <allow_renaming>false</allow_renaming>
+    <pose>#{_x} #{_y} #{_z + 0.2} #{_roll} #{_pitch} #{_yaw}</pose>
+    <world>#{_worldName}</world>
+    <is_performer>true</is_performer>
+    <sdf version='1.6'>
+    <include>
+      <name>#{_name}</name>
+      <uri>#{_modelURI}</uri>
+      <!-- Diff drive -->
+      <plugin filename="libignition-gazebo-diff-drive-system.so"
+              name="ignition::gazebo::systems::DiffDrive">
+        <left_joint>front_left_wheel_joint</left_joint>
+        <left_joint>front_middle_left_wheel_joint</left_joint>
+        <left_joint>rear_middle_left_wheel_joint</left_joint>
+        <left_joint>rear_left_wheel_joint</left_joint>
+        <right_joint>front_right_wheel_joint</right_joint>
+        <right_joint>front_middle_right_wheel_joint</right_joint>
+        <right_joint>rear_middle_right_wheel_joint</right_joint>
+        <right_joint>rear_right_wheel_joint</right_joint>
+        <wheel_separation>#{0.525}</wheel_separation>
+        <wheel_radius>0.129</wheel_radius>
+        <topic>/model/#{_name}/cmd_vel_relay</topic>
```

Comment:
I think velocity and acceleration limits may now be required. For example X1 uses these params

```
      <min_velocity>-1</min_velocity>
      <max_velocity>1</max_velocity>
      <min_acceleration>-3</min_acceleration>
      <max_acceleration>3</max_acceleration>
```
 maybe @acschang  can provide feedback on appropriate values for this vehicle.